package.json

```diff
@@ -135,6 +134,7 @@
     "shelljs": "^0.8.5",
     "stream-browserify": "^3.0.0",
     "stream-http": "^3.2.0",
+    "ts-prune": "^0.10.3",
```

Comment:
do you want to create a script for this? Otherwise it could just be a global install without being in package.json

Reply:
```suggestion
```